Resources/public/js/components/bulk-price/main.js

```diff
@@ -132,36 +135,84 @@ define(['text!suluproduct/components/bulk-price/bulk-price.html'], function(Bulk
                 }
             }.bind(this));
 
+            // special prices
+            var specialPrice = {};
+            specialPrice.currency = {};
+            specialPrice.start = $('#husky-input-js-dateFrom' + this.options.currency.code).val();
```

Comment:
As i can see this, you have to remove the plain id="" attribute and then it should work